### PR TITLE
Add `deposit_amount` feature

### PR DESCRIPTION
**Pull request by syjn99**
Add `deposit_amount` prompt for user to customize the deposit amount. An user can either 1) enter valid deposit amount(256 ~ 16,384) or 2) add a flag like `--deposit_amount=256`. Input should be in Over, not in Gwei or something.

### CODE REVIEW

File: staking_deposit/utils/validation.py [hunk 1]
```diff
@@ -79,7 +80,7 @@ def validate_deposit(deposit_data_dict: Dict[str, Any], credential: Credential)
         return False
 
     # Verify deposit amount
-    if not MIN_DEPOSIT_AMOUNT < amount <= MAX_DEPOSIT_AMOUNT:
+    if not MIN_DEPOSIT_AMOUNT <= amount <= MAX_DEPOSIT_AMOUNT:
```
Comment (schyun9212) [inline staking_deposit/utils/validation.py:83]: How about reusing `validate_deposit_amount`?

File: staking_deposit/utils/validation.py [hunk 2]
```diff
@@ -117,6 +118,19 @@ def validate_int_range(num: Any, low: int, high: int) -> int:
...
         raise ValidationError(load_text(['err_not_positive_integer']))
 
+def validate_deposit_amount(amount: Any) -> int:
+    '''
+    Verifies that `amount` is an `int` and MIN_DEPOSIT_AMOUNT <= num < MAX_DEPOSIT_AMOUNT
+    '''
+    try:
+        amount_int = int(amount)  # Try cast to int
+        assert amount_int == float(amount)  # Check amount is not float
+        amount_int_in_gwei = amount_int * ETH2GWEI
```
Comment (schyun9212) [inline staking_deposit/utils/validation.py]: It seems that `ETH2GWEI` is being used incorrectly in terms of its meaning.

File: staking_deposit/utils/validation.py (hunk 1 above)
Comment (syjn99) [reply staking_deposit/utils/validation.py:83]: I'd like to make a minimum change for existing logic

Comment (syjn99) [reply staking_deposit/utils/validation.py:83]: Also it can be quite messy to wrap this code block with try-catch.

Comment (schyun9212) [reply staking_deposit/utils/validation.py:83]: Is it okay not to check whether the amount is divisible by gwei?

Comment (syjn99) [reply staking_deposit/utils/validation.py:83]: As `amount` is already in gwei, the check would not be needed.

File: staking_deposit/utils/validation.py (hunk 2 above)
Comment (syjn99) [reply staking_deposit/utils/validation.py]: Okay, I will change variable names in these lines.

Comment (syjn99) [reply staking_deposit/utils/validation.py]: e323f8106c51f5d6e16200cdd4866442d18293a6 addresses this issue.

**Pull request merged**